utilities_common/common.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/python -u
```

Comment:
Are you adding this file by accident?

Reply:
Good catch. Yes accidental. Let me remove

Reply:
No this file has been there, for a while. Part of re-factoring as requested by Qi. I just noticed, that recent merge has run_command in cli.py. So I dropped it from common.py.

Is there any moving around, you would prefer ?

Reply:
You are adding this file, it hasn't been there for a while.

Reply:
Yes in this PR, few commits ago. There is no change since last approval, except dropping run_command, as it is now in cli.py.

Do you have any comments for this file?

Reply:
Comments added.

Reply:
Thanks Joe! I will get to each.